packages/salesforcedx-vscode-apex-replay-debugger/src/telemetry/telemetry.ts

```diff
@@ -56,4 +56,32 @@ export class TelemetryService {
       });
     }
   }
+
```

Comment:
It would be good to add a field for error number/code if we are getting one.

Reply:
Unfortunately, we don't get an error code. We would have to create our own codes.

![errorobj](https://user-images.githubusercontent.com/7286437/44678882-4e8d6500-a9ee-11e8-88fb-ec8e04059580.png)

Reply:
Ok, then let's leave that as future work.